src/components/InputTodo.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/prop-types */
+import { useState } from "react";
+
+const InputTodo = ({ addTodoItem }) => {
+  const [title, setTitle] = useState("");
+  const [msg, setMsg] = useState("");
+  const handelChange = (e) => {
+    setTitle(e.target.value);
+  };
+
+  const handelSubmit = (e) => {
+    e.preventDefault();
+    // console.log(title);
+    // console.log(title.trim());
+    if (title.trim()) {
+      addTodoItem(title);
+      setMsg("");
+      setTitle("");
+    } else {
+      setMsg("Please add Item!");
+    }
+  };
+  return (
+    <>
+      <form onSubmit={handelSubmit} className="form-container">
+        <input
+        className="input-text"
+          type="text"
+          placeholder="Add Todo..."
+          value={title}
+          onChange={handelChange}
+        />
+        <button className="input-submit">Submit</button>
```

Comment:
Please add type attribute to the submit button I mean `type="submit"`

Reply:
I'll Fix that, Thanks for your suggestion 👍